common/views/components/ImageViewer/ImageViewer2.js

```diff
@@ -170,11 +170,13 @@ class ImageViewer extends React.Component<ImageViewerProps, ImageViewerState> {
         <Image
           width={this.props.width}
           contentUrl={this.props.contentUrl}
-          lazyload={true}
+          lazyload={false}
```

Comment:
Is this just because we always expect the ImageViewer to be at the top of the page, so might as well load the image straightaway?

Reply:
Yea, without the image, you might as well not be here. We also set the default size to the image size.